### DISCUSSION TITLE
Ran docker set and Lidarr is working, still have some issues/questions

### DISCUSSION THREAD

**Discussion by CoreyCBurton** [RandomNinjaAtk/docker-lidarr-extended#246] [General]
Current Issues
- when I add an artist album to my lidarr, music videos download fine. When it comes to the music itself, it does not seem to download. Will I have to add my own custom indexer or is that included within the script? 
- When the music videos are done downloading along with the music, how is it renamed to fit plex's naming scheme to be recognized

Thank you.

**Comment by RandomNinjaAtk** [comment]
Scripts handles both things... if both are enabled and configured properly
